variables.tf

```diff
@@ -119,8 +119,7 @@ variable "service_endpoints" {
 
 variable "existing_event_notifications_instances" {
   type = list(object({
-    en_instance_id      = string
-    en_region           = string
+    en_crn              = string
```

Comment:
Can we just call this crn instead of en_crn ? The input variable already has the text `event_notifications_instances` in it. Same with:
- `en_integration_name` - > `integration_name`
- `skip_en_auth_policy` -> `skip_iam_auth_policy`